nipype/interfaces/freesurfer/preprocess.py

```diff
@@ -950,14 +989,24 @@ def _is_resuming(self):
     def _format_arg(self, name, trait_spec, value):
         if name == 'T1_files':
             if self._is_resuming():
-                return ''
+                return None
         if name == 'hippocampal_subfields_T1' and \
                 isdefined(self.inputs.hippocampal_subfields_T2):
-            return ''
+            return None
         if all((name == 'hippocampal_subfields_T2',
                 isdefined(self.inputs.hippocampal_subfields_T1) and
                 self.inputs.hippocampal_subfields_T1)):
-            trait_spec.argstr = trait_spec.argstr.replace('T2', 'T1T2')
+            argstr = trait_spec.argstr.replace('T2', 'T1T2')
+            return argstr % value
```

Comment:
This part here was a (minor) bug, in that `trait_spec` is non-ephemeral, so changing `trait_spec.argstr` changes it for all future calls. Unlikely to affect anything except the doctests, but still worth cleaning up.